tests/manage/pv_services/test_no_space_left.py

```diff
@@ -0,0 +1,155 @@
+"""
+Test to verify PVC behavior when full of data with I/O
+"""
+import logging
+import pytest
+import ocs_ci.ocs.exceptions as ex
+
+from ocs_ci.framework.testlib import tier1, E2ETest
+from ocs_ci.ocs import constants
+from ocs_ci.ocs.resources.pod import get_used_space_on_mount_point
+
+log = logging.getLogger(__name__)
+
+
+@pytest.mark.parametrize(
+    argnames=["interface"],
+    argvalues=[
+        pytest.param(
+            *[constants.CEPHBLOCKPOOL],
+            marks=pytest.mark.polarion_id("OCS-852")
+        ),
+        pytest.param(
+            *[constants.CEPHFILESYSTEM],
+            marks=[
+                pytest.mark.polarion_id("OCS-853"),
+                pytest.mark.bugzilla("1745344")
+            ]
+        ),
+    ]
+)
+@tier1
+class TestPVCFullWithIORWO(E2ETest):
+    """
+    Test to verify PVC behavior when full with data
+    """
+    pvc_size_gb = 50
+
+    @pytest.fixture()
+    def base_setup(
+        self, request, interface, access_mode, pvc_factory, pod_factory
+    ):
+        """
+        A setup phase for the test
+        """
+        self.interface = interface
+        self.pvc_obj = pvc_factory(
+            interface=self.interface, size=self.pvc_size_gb,
+            access_mode=access_mode
+        )
+        self.pod_obj = pod_factory(interface=self.interface, pvc=self.pvc_obj)
+
+    @pytest.mark.usefixtures(base_setup.__name__)
+    def test_pvc_no_space_left(self, pod_factory):
+        """
+        Writing data to PVC to reach limit
+        """
+        log.info(f"Running FIO to fill PVC size: {self.pvc_size_gb}")
+        self.pod_obj.run_io(
+            'fs', size=self.pvc_size_gb, io_direction='write', runtime=60
+        )
+        log.info("Waiting for IO results")
+        try:
+            self.pod_obj.get_fio_results()
+        except ex.CommandFailed as cf:
+            if "No space left on device" not in cf.__str__():
+                raise
+            else:
+                used_space = get_used_space_on_mount_point(self.pod_obj)
+                assert used_space == '100%', (
+                    f"The used space is not 100% but {used_space} which means "
+                    f"the device is not full"
+                )
+                log.info(f"FIO succeeded to fill the PVC with data")
+        log.info(f"Deleting the pod and attaching the full PVC to a new pod")
+        self.pod_obj.delete()
+        self.pod_obj = pod_factory(interface=self.interface, pvc=self.pvc_obj)
+        used_space = get_used_space_on_mount_point(self.pod_obj)
+        assert used_space == '100%', (
+            f"The used space is not 100% but {used_space} from the new pod"
+        )
+
+
+@pytest.mark.parametrize(
+    argnames=["interface"],
+    argvalues=[
+        pytest.param(
+            *[constants.CEPHFILESYSTEM],
+            marks=[
```

Comment:
as there is only one permutation, parametrize is not needed

Reply:
done